templates/splitpayments/index.html

```diff
@@ -128,17 +115,30 @@ <h5 class="text-subtitle1 q-my-none">Target Wallets</h5>
     </q-card>
   </div>
 
-  <div class="col-12 col-md-5 q-gutter-y-md">
+  <div class="col-12 col-md-3 q-gutter-y-md">
     <q-card>
       <q-card-section>
         <h6 class="text-subtitle1 q-my-none">
           {{SITE_TITLE}} SplitPayments extension
         </h6>
       </q-card-section>
-      <q-card-section class="q-pa-none">
-        <q-separator></q-separator>
-        <q-list> {% include "splitpayments/_api_docs.html" %} </q-list>
-      </q-card-section>
+      <q-separator></q-separator>
+      <q-card>
+        <q-card-section>
+          <p>
+            Add some wallets to the list of "Target Wallets", each with an
+            associated <em>percent</em>. After saving, every time any payment
+            arrives at the "Source Wallet" that payment will be split with the
+            target wallets according to their percent.
+          </p>
+          <p>This is valid for every payment, doesn't matter how it was created.</p>
+          <p>Target wallets can be any wallet from this same LNbits instance.</p>
```

Comment:
didnt you say it works for any instance now ?

Reply:
I think this should be:

`Targets can be LNBits wallets from this LNBits instance or any valid LNURL or LN Address.`